pyasdf/tags/transform/selector.py

```diff
@@ -0,0 +1,77 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, unicode_literals, print_function
+
+import numpy as np
+from numpy.testing import assert_array_equal
+
+
+from gwcs import selector
```

Comment:
For (4), you would just move the contents of this file to the gwcs project (assuming the schemas still live in asdf-standard).  If we want to put the schemas elsewhere, that will require some minor changes to pyasdf.